libs/shared/ui-components/src/Form/Form.tsx

```diff
@@ -35,7 +40,13 @@ export const Form: FC<TFormProps> = (props) => {
   };
 
   const onSubmit = handleSubmit((formValues): void => {
-    sendFormData(hookUrl, formValues)
+    const paramValues: ParamValues = getParamValues();
+    const combinedValues: FormAndParamValues = combineFormParamValues(
+      formValues,
+      paramValues,
+    );
```

Comment:
Why not do the following?

```suggestion
    const combinedValues = {...formValues, ...paramValues}
```

Reply:
Showing off my novice JS skills here. Thx.